pages/Messages/Overview.md

```diff
@@ -82,7 +82,7 @@ This is what would be posted as a Log event in Ethereum:
 |-------|-------------|------|
```

Comment:
Since we're not emitting individual messages, the entire Variable Announcement Format section can be deleted; it's not specific to serialization, so I can include it as part of my Batch spec changes instead if you like.